Updates to articles to conform to some documentation style. (#344)
swiftlang/swift-testing: Sources/Testing/Testing.docc/ParameterizedTesting.md

> # Parameterizing testing

Reviewer: "Parameterized testing" is a noun phrase here, sort of. It's a distinct concept (or subtopic) of "test" in general. I won't dwell on it, but it might be worth finding a way to preserve the term of art "parameterized testing" here. (Or maybe we just give up on it as a term of art entirely?)

Reviewer: Maybe?

- "Running a test with multiple inputs"
- "Testing with different input values"

Something like that? Just drop the word "parameterized" so we don't confuse it for a verb later (it's an adverb in this position.)

or more inputs, the corresponding diagnostics can clearly indicate which inputs
need to be examined.

## Parameterize over the cases of an enumeration

> In the example above, we hard-coded the list of `Food` cases to test. If `Food`

Reviewer 2: ```suggestion
The previous example includes a hard-coded list of `Food` cases to test. If `Food`
```

> is an enumeration conforming to `CaseIterable`, we can instead write:

Reviewer 2: ```suggestion
is an enumeration that conforms to `CaseIterable`, you can instead write:
```

Avoid using first person voice.

> This way, if a new case is added to the `Food` enumeration, it will
> automatically be tested by this test function.

Reviewer 2: ```suggestion
This way, if a new case is added to the `Food` enumeration, it is
automatically tested by this function.
```

}
```

> - Note: Large ranges such as `0 ..< .max` may take an excessive amount of

Reviewer: We want to emphasize that galactically-sized input sequences are a bad idea. :)

> - Note: Very large ranges such as `0 ..< .max` may take an excessive amount of

Reviewer 2: ```suggestion
- Note: Large ranges such as `0 ..< .max` may take an excessive amount of
```

> If any argument of a test case does not meet one of the above requirements, then

Reviewer 2: ```suggestion
If any argument of a test case doesn't meet one of the above requirements, then
```
